de-lib/src/main/java/org/iplantc/de/apps/client/views/ReactAppDetailsView.java

```diff
@@ -0,0 +1,17 @@
+package org.iplantc.de.apps.client.views;
+
+import org.iplantc.de.apps.client.AppDetailsView;
+
+import com.google.web.bindery.autobean.shared.Splittable;
+
+import jsinterop.annotations.JsType;
+
+/**
+ * @author psarando
+ */
+@JsType(isNative = true, namespace = "CyVerseReactComponents.apps", name = "details")
+public final class ReactAppDetailsView {
```

Comment:
The methods in this `ReactAppDetailsView ` class replace the native methods in `AppDetailsViewImpl` that were calling the React lib functions.

Note the annotation properties `namespace` and `name` configure this class so that calling the following in Java:
```java
ReactAppDetailsView.renderToolDetails(...)
```
translates to this in JS:
```javascript
CyVerseReactComponents.apps.details.renderToolDetails(...)
```